### ISSUE TITLE
add hyperparameters as additional variables to simulations

### ISSUE THREAD

**Issue by wkvong** [alexatartaglini/alcove#7]
In addition to the things we're already varying (model, image set, loss function etc.), we'll also want to modify the hyperparameters of ALCOVE (c, phi, lr_association and lr_attn) and add these to the `itertools.product()` call at the beginning to loop over. I'll edit this issue with sensible ranges for each of these values when I have some more time!

**Comment by wkvong** [comment]
here's some tentative ranges for the different hyperparameters to try out:
- c: 0.5 - 10.0 in 0.5 increments, i.e. [0.5, 1.0, 1.5, ..., 10.0]
- phi: 0.25 -5.0 in 0.25 increments, i.e. [0.25, 0.5, ..., 5.0]
- lr_attn: 0.001 to 0.010 in 0.001 increments, i.e. [0.001, 0.002, 0.003, ..., 0.01] (might want to experiment here)
- lr_assoc: 0.005 to 0.05 in 0.005 increments, i.e. [0.005, 0.010, 0.015, ..., 0.05]
